addons/context_actions/functions/fnc_repairVehicles.sqf

```diff
@@ -0,0 +1,22 @@
+/*
+ * Author: 3Mydlo3
+ * Repairs units in given selection.
+ *
+ * Arguments:
+ * N: Selected Objects <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [vehicle1, vehicle2, vehicle3] call zen_context_actions_fnc_repairVehicles
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private _vehicles = _this select {alive _x && {_x isKindOf "AllVehicles" && {!(_x isKindOf 'Man')} && {damage _x > 0}}};
```

Comment:
damage check is cheaper than the isKindOf's, should be ordered by cost.

Reply:
Where can I check these costs? Is there any other way than trying myself with `BIS_fnc_codePerformance`?

Reply:
It's mostly knowledge/experience and manual benchmarking, eg. `isKindOf` is related to configs and config lookups are known to be slow so it makes sense that simple damage check should be faster as it's just an object state.

btw: use `diag_codePerformance` instead of BIS fnc.